ansible/roles/cluster/tasks/main.yaml

```
@@ -0,0 +1,27 @@
+  ansible.builtin.template:
+    src: ../../templates/merged_cluster.yaml.j2
+    dest: '/tmp/cluster.yaml'
+    mode: '0644'
+  vars:
+    ssh_pub_key: "{{ ssh_pub_key_file.content | b64decode }}"
+
+- name: 'Create EKS-A cluster'
+  become: true
+  ansible.builtin.command:
+    cmd: 'eksctl anywhere create cluster -f /tmp/cluster.yaml --hardware-csv /tmp/hardware.csv'
+  run_once: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)